mods/cnc/chrome/editor.yaml

```diff
@@ -201,6 +205,7 @@ Container@EDITOR_ROOT:
 
 Container@EDITOR_WORLD_ROOT:
 	Logic: LoadIngamePerfLogic, MapEditorLogic
+		ChangeZoomKey: TogglePixelDouble
```

Comment:
only side note: with latest nice consistency (StopMusicKey: StopMusic,..) this difference is now more visible

Reply:
This one is intentional: the editor changes the zoom between a range of steps, but happens to be bound to the more standard pixel doubling hotkey.